.circleci/build.sh

```diff
@@ -73,6 +73,7 @@ rm -rf vision
 # PyTorch CI
 git clone https://github.com/pytorch/vision --quiet
 pushd vision
+git reset --hard c7191d56ff639f3abd17339f9f0b47ded7a28417
```

Comment:
What is the problem here?

Reply:
Recent breakage (last commit, hours ago), it seems:

```
Traceback (most recent call last):
  File "test/test_train_mnist.py", line 124, in test_accurracy
    self.assertGreaterEqual(train_mnist(), FLAGS.target_accuracy)
  File "test/test_train_mnist.py", line 68, in train_mnist
    transforms.Normalize((0.1307,), (0.3081,))
  File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/mnist.py", line 68, in __init__
    self.download()
  File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/mnist.py", line 143, in download
    download_url(url, root=self.raw_folder, filename=filename, md5=None)
  File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/utils.py", line 73, in download_url
    reporthook=gen_bar_updater(tqdm())
TypeError: __init__() missing 1 required positional argument: 'total'
```